.github/workflows/reusable-integration-tests.yml

```diff
@@ -35,7 +35,7 @@ on:
       module_branch:
         required: false
         type: string
-        default: "master"
```

Comment:
Have you checked that all repositories that were using that reusable workflow are using `main` as default branch?